panda/src/gles2gsg/CMakeLists.txt

```diff
@@ -13,10 +13,13 @@ set(P3GLES2GSG_SOURCES
 )
 
 composite_sources(p3gles2gsg P3GLES2GSG_SOURCES)
-add_component_library(p3gles2gsg SYMBOL BUILDING_PANDAGLES2
-  ${P3GLE2SGSG_HEADERS} ${P3GLES2GSG_SOURCES})
-target_link_libraries(p3gles2gsg p3glstuff panda
-  PKG::GLES2)
+add_component_library(p3gles2gsg SYMBOL BUILDING_PANDAGLES2 ${P3GLE2SGSG_HEADERS} ${P3GLES2GSG_SOURCES})
+
+if (ANDROID)
+    target_link_libraries(p3gles2gsg p3glstuff panda PKG::GLES2)
+else()
+    target_link_libraries(p3gles2gsg p3glstuff panda PKG::GLES2)
+endif()
```

Comment:
These cases are identical?  This change seems to change nothing

Reply:
indeed it's a leftover